modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -236,6 +237,7 @@ TensorMechanicsApp::registerObjects(Factory & factory)
   registerMaterial(ComputeReturnMappingStress);
   registerMaterial(RecomputeRadialReturn);
   registerMaterial(RecomputeRadialReturnIsotropicPlasticity);
+  registerMaterial(RecomputeRadialReturnIsotropicPowerLawHardening);
```

Comment:
Sorry, this is not a new record in object name length. `MultiAppVariableValueSamplePostprocessorTransfer` still holds that one with 48 characters (yours only has 47... wanna try again?).